node-require/calculate.js

```diff
@@ -0,0 +1,20 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
+
+switch(process.argv[3]){
+  case 'plus':
+    console.log(add.add(Number(process.argv[2]), Number(process.argv[4])))
+    break;
+  case 'minus':
+    console.log(subtract.subtract(Number(process.argv[2]), Number(process.argv[4])))
+    break;
+  case 'over':
+    console.log(divide.divide(Number(process.argv[2]), Number(process.argv[4])))
```

Comment:
If you assign your function to the `exports` property, you can eliminate the need to do `divide.divide`

```js
module.exports = divide;
```

```js
const divide = require('./divide');
```